config/v1/types_infrastructure.go

```diff
@@ -144,6 +148,18 @@ type GCPPlatformStatus struct {
 	Region string `json:"region"`
 }
 
+// OpenStackPlatformStatus holds the current status of the OpenStack infrastructure provider.
+type OpenStackPlatformStatus struct {
+	// apiIntIP is an IP address managed by the OpenStack provider
+	// backing the internal `api-int` record.
```

Comment:
can you describe what this `api-int` record is used for?  kube-apiserver?  Some openstack informational URL?

Reply:
Yeah it's the kube API for the internal services